examples/oit_reconstruction.py

```diff
@@ -0,0 +1,367 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Example of shape-based image reconstruction
+using optimal information transformation.
+"""
+
+# Initial setup
+import numpy as np
+import numba
+import matplotlib.pyplot as plt
+import time
+import ddmatch
```

Comment:
We cannot import `ddmatch` since it's not an official package. And we cannot distribute it with ODL either. Unfortunately, we need to replicate its functionality, at least the part we need for our purposes. As far as I can see, it shouldn't be hard.